packages/block-library/src/post-time-to-read/edit.js

```diff
@@ -6,27 +6,67 @@ import clsx from 'clsx';
 /**
  * WordPress dependencies
  */
-import { _x, _n, sprintf } from '@wordpress/i18n';
-import { useMemo } from '@wordpress/element';
+import { __, _x, _n, sprintf } from '@wordpress/i18n';
+import { useMemo, useEffect } from '@wordpress/element';
 import {
 	AlignmentControl,
 	BlockControls,
+	InspectorControls,
 	useBlockProps,
+	store as blockEditorStore,
 } from '@wordpress/block-editor';
+import {
+	RangeControl,
+	ToggleControl,
+	__experimentalToolsPanel as ToolsPanel,
+	__experimentalToolsPanelItem as ToolsPanelItem,
+} from '@wordpress/components';
 import { __unstableSerializeAndClean } from '@wordpress/blocks';
 import { useEntityProp, useEntityBlockEditor } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
 import { count as wordCount } from '@wordpress/wordcount';
 
 /**
- * Average reading rate - based on average taken from
- * https://irisreading.com/average-reading-speed-in-various-languages/
- * (Characters/minute used for Chinese rather than words).
+ * Internal dependencies
  */
-const AVERAGE_READING_RATE = 189;
+import { useToolsPanelDropdownMenuProps } from '../utils/hooks';
+
+function PostTimeToReadEdit( {
+	attributes,
+	setAttributes,
+	clientId,
+	context,
+} ) {
+	/**
+	 * Average reading rate - based on average taken from
+	 * https://irisreading.com/average-reading-speed-in-various-languages/
+	 * (Characters/minute used for Chinese rather than words).
+	 */
+	const AVERAGE_READING_RATE = 189;
+
+	const { blockWasJustInserted } = useSelect(
+		( select ) => {
+			const { wasBlockJustInserted } = select( blockEditorStore );
+
+			return {
+				blockWasJustInserted: wasBlockJustInserted( clientId ),
+			};
+		},
+		[ clientId ]
+	);
 
-function PostTimeToReadEdit( { attributes, setAttributes, context } ) {
-	const { textAlign } = attributes;
+	// When the block is first inserted, default to displaying as a range.
+	useEffect( () => {
+		if ( blockWasJustInserted ) {
+			setAttributes( {
+				displayAsRange: true,
+			} );
+		}
+	}, [ blockWasJustInserted ] );
```

Comment:
Now I understand what this effect wants to do.

So you want to enable `displayAsRange` only in newly inserted blocks? Do we need to enable `displayAsRange` by default?

Reply:
Once the block is inserted you can still turn the range off. I think this presents a good compromise - it encourages people to use the more accessible version but doesn't force it for those who really want to use the simpler version.